### ISSUE TITLE
Conv 1D block with weights initialized by me and controllable gradients

### ISSUE THREAD

**Issue by svj1991** [apache/incubator-mxnet#15047]
Hi,
I’m a bit new to Gluon and mxnet, I’m sorry if my question is too trivial.
I was trying to implement a Conv 1D operation on a speech waveform.

I have the following,

N = 512
hp = 16
re = nn.Conv1D( in_channels=1, channels=N, kernel_size=N, padding=N//2, strides=hp, groups=1, use_bias=False)

Now I have a ndarray 'br' of size (N,1,N) and I wish to initialize the Conv1D layer re with this 'br' matrix. Also, I’d like to have the control to make this layer trainable or fixed, essentially setting the gradient required to True or False. I have been struggling to find how to do this in the documentation. Could anyone help?

Thanks.

**Comment by zachgk** [comment]
For questions, please submit them on the MXNet discussion forum (https://discuss.mxnet.io) where it will get a wider audience and allow others to learn as well.
 I'm closing this issue now in favor of the discussion forum issue you will file, please feel free to re-open if closed in error.
 Thanks!
